tests/framework/installer/install_helper.go

```diff
@@ -271,18 +275,9 @@ func (h *InstallHelper) UninstallRookFromK8s(namespace string, helmInstalled boo
 	_, err = k8sHelp.DeleteResource([]string{"namespace", namespace})
 	assert.NoError(h.T(), err, "cannot delete namespace %s err -> %+v", namespace, err)
 
-	isRookUninstalled := k8sHelp.WaitUntilPodInNamespaceIsDeleted("rook-ceph-mon", namespace)
-	isNameSpaceDeleted := k8sHelp.WaitUntilNameSpaceIsDeleted(namespace)
 	h.k8shelper.Clientset.RbacV1beta1().ClusterRoleBindings().Delete("anon-user-access", nil)
 
-	assert.True(h.T(), isRookUninstalled, "make sure rook is uninstalled")
-	assert.True(h.T(), isNameSpaceDeleted, "make sure rook is uninstalled")
-
-	if isRookUninstalled && isNameSpaceDeleted {
-		logger.Infof("Rook cluster %s uninstalled successfully", namespace)
-		return
-	}
-	logger.Errorf("Rook cluster %s not uninstalled", namespace)
+	logger.Errorf("Rook cluster %s uninstalled", namespace)
```

Comment:
this should probably not be Error level anymore

Reply:
good catch